source/stylesheets/refills/unstyled/_navigation.scss

```diff
@@ -0,0 +1,82 @@
+header.navigation {
+  $large-screen: em(860) !default;
+  $large-screen: $large-screen;
+
+  // Mobile view
+  .navigation-menu-button {
+    display: block;
+    float: right;
+    margin: 0;
+    padding-top: 0.5em;
+
+    @include media ($large-screen) {
+      display: none;
+    }
+  }
+
+  // Nav menu
+  .navigation-wrapper {
+    @include clearfix;
+    position: relative;
+  }
+
+  .logo {
+    float: left;
+
+    img {
```

Comment:
If I change this hound will complain of nesting too deep. How do we deal with this @kaishin @tysongach?